### ISSUE TITLE
Expression 'undefined' used with directive 'nitAdvancedSearchbox' is non-assignable!

### ISSUE THREAD

**Issue by vaibhav93** [dnauck/angular-advanced-searchbox#34]
The default template appears without search Params and this error is thrown. The css and tpls files are loaded correctly.

**Comment by vaibhav93** [comment]
Managed to resolve it.
```
            scope: {
                model: '=ngModel',
                parameters: '=',
                parametersLabel: '@',
                parametersDisplayLimit: '=',
                placeholder: '@',
                searchThrottleTime: '='
            },
```
should be changed to 
            ```
scope: {
                model: '=ngModel',
                parameters: '=',
                parametersLabel: '@',
                parametersDisplayLimit: '=?',
                placeholder: '@',
                searchThrottleTime: '=?'
            },
```

This is done since parametersDisplayLimit and searchThrottleTime bindings are optional.
Reference :
https://docs.angularjs.org/error/$compile/nonassign
https://docs.angularjs.org/api/ng/service/$compile#directive-definition-object

Thanks,
Nice plugin btw

**Comment by jarandaf** [comment]
You seem right, feel free to create a PR.

**Comment by dnauck** [comment]
Thanks for the bug report, please test the latest build

**Comment by jarandaf** [comment]
@dnauck that was fast, thank you!

**Comment by dnauck** [comment]
Fix released with version 2.1.0
